webauthn.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 2.4
 name:          webauthn
-version:       0.0.2
+version:       0.0.3
```

Comment:
Versioning is my job so you don't have to update it in a PR. Btw version Y (of X.Y.Z) should have been updated because it changes the API.